org.splash.Splash.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Your manifest doesn't have `cleanup` array, meaning that various development files are left in the final app.